.gitignore

```
@@ -3,6 +3,7 @@ lerna-debug
 npm-debug
 .DS_Store
 /dist
+website/docs/reference/api/unleash/sidebar.js
```

Oh, I get it now. It was in the docs .gitignore and _not_ the main .gitignore. Because .gitignores don't stack, it needs to be here too?

Suggest changing it to this for consistency, though. Might be more sidebars there later.

```suggestion
website/docs/reference/api/**/sidebar.js
```

---

Or wait ... **do** gitignores stack?

---

Sure good call

---

I think that depends on how you have them configured. It might be a my git being a git issue but I think if it keeps coming back on my machine it's worth adding it to the base config anyway, it's going to drive someone else mad too

---

Could also be the way I'm adding my files, but same issue - someone's going to get hit by this